autoload/miniSnip.vim

```diff
@@ -62,7 +62,8 @@ function! s:directories() abort
 endfunction
 
 function! s:findSnippetFile() abort
-  let cword = matchstr(getline('.'), '\v\f+%' . col('.') . 'c')
+  let expandpattern = s:getVar("expandpattern")
+  let cword = matchstr(getline('.'), '\v' . expandpattern . '+%' . col('.') . 'c')
```

Comment:
The `\v` really needs to be after `expandpattern`, because it modifies how the pattern is interpreted.

Reply:
Ok. Will check.

Reply:
```
'\v\f+%' -> expand + '\v+%'
```

not?

```
'\v\f+%' -> '\v' + expand + '+%'
```

With your suggestion
![obraz](https://user-images.githubusercontent.com/240986/183489266-be809e0e-e1b1-4eca-a883-9cdaad4f4ff9.png)

I need more info, how it should be interpreted.

Reply:
Ok. Updated.